scripts/fixtures-tool.js

```diff
@@ -0,0 +1,78 @@
+import { glob } from "glob";
```

Comment:
Can you call this file `cleanup-fixtures`? Also, can we run it on CI, failing if it requires any changes?